modules/google-analytics.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+/**
+ * Module Name: Google Analytics
+ * Module Description: MODULE DESCRIPTION COPY.
+ * First Introduced: 4.4
+ * Sort Order: 37
+ * Requires Connection: No
+ * Auto Activate: No
+ * Feature: Engagement
+ * Additional Search Queries: webmaster, google, analytics, console
+ */
+
+/**
+ * Removes the Google Analytics plugin settings page
+ */
+function removeSettingsPage() {
+	remove_submenu_page( 'options-general.php', 'wp-google-analytics' );
+}
+
+/**
+ * Does the site have a Jetpack plan attached to it that includes Google Analytics?
+ *
+ * @return bool
+ */
+function isGoogleAnalyticsIncludedInJetpackPlan() {
```

Comment:
Instead of creating a new function, it might be worth keeping an eye on #5587, where we're creating a more standard way to check for the status of a site.